### PR TITLE
gh-155 Set AddElementsFromHdfs builder methods to public

### PR DESCRIPTION
**Pull request by james010101101**

### CODE REVIEW

**Comment by p013570** [comment]
We have had a few problems with builders now - perhaps we should add the following method to the OperationTest interface:
```java
@Test
void shouldBuildOpeartion();
```

Then we should write a test for all operation builders to check all the builder methods are available, public, return the correct builder and set the correct field on the operation.

File: accumulo-store/src/main/java/gaffer/accumulostore/operation/hdfs/impl/SampleDataForSplitPoints.java [hunk 1]
```diff
@@ -116,17 +117,17 @@ public Builder proportionToSample(final float proportionToSample) {
         }
 
         @Override
-        protected Builder inputPaths(final List<Path> inputPaths) {
+        protected Builder inputPaths(final List<String> inputPaths) {
```
Comment (p013570) [inline accumulo-store/src/main/java/gaffer/accumulostore/operation/hdfs/impl/SampleDataForSplitPoints.java]: Should be changed to public

File: accumulo-store/src/main/java/gaffer/accumulostore/operation/hdfs/impl/SampleDataForSplitPoints.java [hunk 2]
```diff
@@ -116,17 +117,17 @@ public Builder proportionToSample(final float proportionToSample) {
...
             return (Builder) super.inputPaths(inputPaths);
         }
 
         @Override
-        protected Builder addInputPaths(final List<Path> inputPaths) {
+        protected Builder addInputPaths(final List<String> inputPaths) {
```
Comment (p013570) [inline accumulo-store/src/main/java/gaffer/accumulostore/operation/hdfs/impl/SampleDataForSplitPoints.java]: Should be changed to public

File: accumulo-store/src/main/java/gaffer/accumulostore/operation/hdfs/impl/SampleDataForSplitPoints.java [hunk 3]
```diff
@@ -116,17 +117,17 @@ public Builder proportionToSample(final float proportionToSample) {
...
             return (Builder) super.addInputPaths(inputPaths);
         }
 
         @Override
-        protected Builder addInputPath(final Path inputPath) {
+        protected Builder addInputPath(final String inputPath) {
```
Comment (p013570) [inline accumulo-store/src/main/java/gaffer/accumulostore/operation/hdfs/impl/SampleDataForSplitPoints.java]: Should be changed to public

**Comment by james010101101** [comment]
Will be merged with pull request gh-161

**Pull request closed without merging**